src/main/java/org/gridsuite/filter/server/FilterService.java

```diff
@@ -331,4 +337,23 @@ public List<FilterEquipments> exportFilters(List<UUID> ids, Network network, Set
             .map(filter -> filter.toFilterEquipments(FilterServiceUtils.getIdentifiableAttributes(filter, network, filterLoader)))
             .toList();
     }
+
+    private Map<String, Object> getCyclicFilterNames(String userId, FilterCycleException exception) {
+        try {
```

Comment:
Maybe put the try/catch in the calling method so that we throw the exception only once, with cyclicFilterNames if we succeeded and with nothing if not